extensions/curator/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "$CRON /usr/bin/curator  --config ${CONFIG_FILE} ${COMMAND}" >>/etc/crontabs/root
+
+crond -f -d 8 -l 8
```

Comment:
You defined `CRON: 0 0 * * *` in the config file, so do we need to run `crond` at all?

If yes: `crond` doesn't handle SIGTERM properly. Consider monitoring it using [`tini`](https://pkgs.alpinelinux.org/package/edge/community/x86/tini). eg.
```sh
tini -- crond ...
```

Reply:
It really depends if curator needs to run automatically or manually.
As for tini, I need to do basic learning/reading.

Reply:
tini is really dumb, it doesn't have any option. All you have to do is call it as I suggested and it will take care of starting `crond` as a child process and pass it all OS signals received by the container.

As a bonus, it will reap zombie processes that may end up staying around forever in your container :)

The best reference I could this about is https://github.com/krallin/tini/blob/master/README.md

You will also need to enable the community repo: https://wiki.alpinelinux.org/wiki/Enable_Community_Repository

```dockerfile
RUN echo 'http://dl-6.alpinelinux.org/alpine/edge/community' > /etc/apk/repositories \
  && apk -U add --no-cache tini python...
```